.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -0,0 +1,50 @@
+name: 🚀 Feature Request
+description: Suggest an idea or a feature for this SDK
+labels: [ "feature request" ]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this feature request!
```

```suggestion
      value: Thanks for taking the time to fill out this feature request!
```
There's no need for the pipe if it's a single line.